core/src/rpc_pubsub.rs

```diff
@@ -340,13 +340,18 @@ mod tests {
         // Test signature confirmation notification
         let response = robust_poll_or_panic(receiver);
         let expected_res: Option<transaction::Result<()>> = Some(Ok(()));
-        let expected_res_str =
-            serde_json::to_string(&serde_json::to_value(expected_res).unwrap()).unwrap();
-        let expected = format!(
-            r#"{{"jsonrpc":"2.0","method":"signatureNotification","params":{{"result":{},"subscription":0}}}}"#,
-            expected_res_str
-        );
-        assert_eq!(expected, response);
+        let expected = json!({
+           "jsonrpc": "2.0",
+           "method": "signatureNotification",
+           "params": {
+               "result": {
+                   "context": { "slot": 0},
```

Comment:
```suggestion
                   "context": { "slot": 0 },
```
Total nit. Same on all tests in this file and in rpc_subscriptions.rs